doc/whats_new/_contributors.rst

```diff
@@ -169,3 +169,5 @@
 .. _Roman Yurchak: https://github.com/rth
 
 .. _Hanmin Qin: https://github.com/qinhanmin2014
+
+.. _Stephane Couvreur: https://scouvreur.github.io/
```

Comment:
We usually only add core developers and very regular contributors here

Reply:
Apologies I had no idea, fixed in commit [#57b1d0d](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/57b1d0de937116302af17545717b435df38f6bb0)